tripper/triplestore.py

```diff
@@ -539,6 +539,23 @@ def list_databases(cls, backend: str, **kwargs):
     # interfaces to the triples(), add_triples() and remove() methods
     # implemented by all backends.
 
+    prefer_sparql = property(
+        fget=lambda self: getattr(self.backend, "prefer_sparql", None),
+        doc=(
+            "Whether the backend prefer SPARQL over the triples() interface. "
+            "Is None if not specified by the backend."
+            "\n\n"
+            "Even though Tripper requires that the Triplestore.triples() is "
+            "implemented, we have to use Triplestore.query() for some "
+            "backends in some cases (like fuseki when working on RDF lists) "
+            "because of the issue with blank nodes getting renamed. "
+            "\n\n"
+            "The purpose of this property is to allow a user of tripper to "
+            "automatically select the most appropriate interface depending "
+            "on the current backend."
```

Comment:
```suggestion
            "Even though Tripper requires that the Triplestore.triples() is "
            "implemented, Triplestore.query() must be used for some "
            "backends in specific cases (like fuseki when working on RDF lists) "
            "because of how blank nodes are treated. "
            "\n\n"
            "The purpose of this property is to allow a user of tripper to "
            "automatically select the most appropriate interface depending "
            "on the current backend."
```